test/php/library/shared/communicate/CommunicateTest.php

```diff
@@ -38,101 +37,6 @@ public static function setUpBeforeClass(): void
         self::$environ = new MongoTestEnvironment();
     }
 
-    public function testCommunicateToUser_NoFromAddress_Exception()
```

Comment:
we need to keep the tests since we still need the CommunicateToUser method for sending emails to users.  The tests just need to be fixed up after removing communicate_via

Reply:
I'd like to confirm that for sure if you don't mind, I really think these were related only to project update type comms.

Reply:
You are right - we don't need CommunicateToUser.  Thanks for helping me see that.